specification/apimanagement/data-plane/ApiManagement/readme.md

```diff
@@ -0,0 +1,137 @@
+# API Management Direct Control Plane Client
+    
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for Direct Control Plane SDK for API Management.
+
+
+
+---
+## Getting Started 
+To build the SDK for API Management, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
+
+> `autorest`
+
+To see additional help and options, run:
+
+> `autorest --help`
+---
+
+## Configuration
+
+
+
+### Basic Information 
+These are the global settings for the API Management Direct Control Plane API.
+
+``` yaml
+openapi-type: default
```

Comment:
why this is default, it probably should be data-plane, could you check if there's any services have data-plane as the value?